hillrange/ckeditor/0.1/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+    "bundles": {
+        "Hillrange\\CKEditor\\HillrangeCKEditorBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "composer-scripts": {
+        "cache:clear": "symfony-cmd",
+        "ckeditor:install": "symfony-cmd",
+        "assets:install --symlink --relative %PUBLIC_DIR%": "symfony-cmd"
```

Comment:
This is done by other bundle. Please remove. 

Make sure your bundle has a dependency on symfony/asset

Reply:
Thanks Tobias,  This has all been done in version 0.1.07.  The install of ckeditor remains as it is unique to this bundle.

Reply:
Im not sure what you mean. 
`assets:install` and `cache:clear` is not unique to this bundle. They should be removed from this recipe. 

The `ckeditor:install` command should remain.